src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,9 +1,60 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.CsvFileReaderService;
+import core.basesyntax.service.CsvFileWriterService;
+import core.basesyntax.service.ReportGeneratorService;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.impl.CsvFileReaderServiceImpl;
+import core.basesyntax.service.impl.CsvFileWriterServiceImpl;
+import core.basesyntax.service.impl.ReportGeneratorServiceImpl;
+import core.basesyntax.service.impl.ShopServiceImpl;
+import core.basesyntax.strategy.BalanceOperation;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.PurchaseOperation;
+import core.basesyntax.strategy.ReturnOperation;
+import core.basesyntax.strategy.SupplyOperation;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
 public class HelloWorld {
-    // HINT: In the `public static void main(String[] args)` it is better to create instances of your classes, 
-    // and call their methods, but do not write any business logic in the `main` method!
+    public static void main(String[] args) {
+        CsvFileReaderService csvFileReaderService = new CsvFileReaderServiceImpl();
+        CsvFileWriterService csvFileWriterService = new CsvFileWriterServiceImpl();
+        ReportGeneratorService reportGeneratorService = new ReportGeneratorServiceImpl();
+
+        Map<FruitTransaction.Operation, OperationHandler> operationHandlers = new HashMap<>();
+        operationHandlers.put(FruitTransaction.Operation.BALANCE, new BalanceOperation());
+        operationHandlers.put(FruitTransaction.Operation.SUPPLY, new SupplyOperation());
+        operationHandlers.put(FruitTransaction.Operation.PURCHASE, new PurchaseOperation());
+        operationHandlers.put(FruitTransaction.Operation.RETURN, new ReturnOperation());
+
+        ShopService shopService = new ShopServiceImpl(operationHandlers);
+
+        try {
+            List<String> inputReport = csvFileReaderService.readFromFile(
+                    "src/main/resources/reportToRead.csv");
```

Comment:
Use a relative path without specifying `src/main/resources/` for resources, as this will be the default location when running the application from a JAR file or another environment. You can use just `reportToRead.csv`.